src/org/zaproxy/zap/extension/openapi/generators/BodyGenerator.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2017 The ZAP Development Team
+ *  
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ */
+package org.zaproxy.zap.extension.openapi.generators;
+
+import io.swagger.models.properties.Property;
+import io.swagger.models.properties.RefProperty;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class BodyGenerator {
+
+    @SuppressWarnings("serial")
+    private static final Map<String, String> SYNTAX = Collections.unmodifiableMap(new HashMap<String, String>() {
+        {
+            put("object_begin", "{");
+            put("object_end", "}");
+            put("array_begin", "[");
+            put("array_end", "]");
+            put("property_container", "\"");
+            put("inner_separator", ":");
+            put("outer_separator", ",");
+        }
+    });
+
+    public static String generate(String name, boolean isArray) {

```

Comment:
Yes, with <https://firefox.settings.services.mozilla.com/v1/__api__> the request `POST	https://firefox.settings.services.mozilla.com/v1/buckets`.